lib/plugins/aws/index.js

```diff
@@ -59,9 +59,19 @@ class SDK {
       // req.on('send', function (r) {console.log(r)});
 
       return new BbPromise((resolve, reject) => {
-        req.send((err, data) => {
+        req.send((errParam, data) => {
+          const err = errParam;
           if (err) {
-            reject(err);
+            if (err.message === 'Missing credentials in config') {
+              const errorMessage = [
+                '"default" profile was not found in ~/.aws/credentials.',
```

Comment:
Would not specifically mention the default profile in the file. We should only tell them that the provider configuration wasn't found and that they can check our documentation to see how to set it up. Because they might want to set it up differently and don't know what the default profile is